test/models/category_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class CategoryTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
- [ ] Kindly ensure all your test files have test in them , its not good practise to have empty test file.